### PR TITLE
Update GIJoe store heroes

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by Lukewh** [comment]
## QA

- Go to the store in gisf - you should have three buttons at the top to view different information
- Do the same in gijoe and you should get the same boxes with 'view' buttons that link to search results.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7314/

File: jujugui/static/gui/src/app/components/store/test-store.js [hunk 1]
```diff
@@ -144,4 +149,19 @@ describe('Store', function() {
       }
     });
   });
+
+  it('shows different hero links in gijoe', () => {
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/store/test-store.js]: s/var/const/ everywhere in this test.

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**